packages/as-types/assembly/safeMath.ts

```diff
@@ -26,8 +26,11 @@ export class SafeMath {
    * @throws if the operation results in a number bigger than u64.MAX_VALUE.
    */
   static add(a: u64, b: u64): u64 {
+    assert(a + b >= a, 'SafeMath: addition overflow');
```

Comment:
This is the real assertion, but not well presented IMHO.
What you want is that:
a + b < u64.MAX_VALUE
<=> b < u64.MAX_VALUE - a (we substract by a on the left and on the right of the inequality as a >= 0)

So you assertiion should be :
b < u64.MAX_VALUE - a
```suggestion
    assert(b < u64.MAX_VALUE - a, 'SafeMath: addition overflow');
```